lib/dolly/request.rb

```diff
@@ -88,6 +92,10 @@ def values_to_json hash
     def full_path resource
       "/#{database_name}/#{resource}"
     end
+
+    def attachment_path resource, attachment_name
+      "#{full_path(resource)}/#{attachment_name}"
```

Comment:
probably out of scope for this PR, but it seems like `full_path` should be renamed to `resource_path`, since it really isn't the full path anymore.